lib/vagrant-vmck/config.rb

```diff
@@ -5,19 +5,22 @@ class Config < Vagrant.plugin('2', :config)
       attr_accessor :private_key_path
       attr_accessor :memory
       attr_accessor :cpus
+      attr_accessor :image_path
 
       def initialize
         @vmck_url = UNSET_VALUE
         @private_key_path = UNSET_VALUE
         @memory = UNSET_VALUE
         @cpus = UNSET_VALUE
+        @image_path = UNSET_VALUE
       end
 
       def finalize!
         @vmck_url = nil if @vmck_url == UNSET_VALUE
         @private_key_path = nil if @private_key_path == UNSET_VALUE
         @memory = 512 if @memory == UNSET_VALUE
         @cpus = 1 if @cpus == UNSET_VALUE
+        @image_path = 'imgbuild-master.qcow2.tar.gz' if @image_path == UNSET_VALUE
```

Comment:
Please create an issue to make `image_path` mandatory. We'll have to add it to all our vagrantfiles too.